### ISSUE TITLE
CreateBucketTask ignores region

### ISSUE THREAD

**Issue by cagataygurturk** [classmethod/gradle-aws-plugin#48]
This task always creates bucket in us-east-1.

**Comment by cagataygurturk** [comment]
Thanks for the fix, for when should we expect the next release?

**Comment by glebsts** [comment]
historical note: was merged, can be closed.

**Comment by cagataygurturk** [comment]
I know it's merged but since 14 July you haven't released any code. When will you release? Thanks.

**Comment by cagataygurturk** [comment]
Also, the plugin already has a global region value 

```
apply plugin: "jp.classmethod.aws"
    aws {
        region = "eu-west-1"
    }
```

I believe that if region value is not given in CreateBucketTask task, this global value should be used instead of us-east-1.

**Comment by glebsts** [comment]
Thank you @cagataygurturk 
a) http://docs.aws.amazon.com/general/latest/gr/rande.html#s3_region:
> For accounts created on or after March 8, 2013, the default region is us-west-2; for older accounts, the default region is us-east-1.

So default value corresponds to Amazon default, and that is expected.

b) Not sure where is aws.region used currently, but CreateBucketTask did not use it before PR, it used AWS default from SDK (see previous note).

**Comment by cagataygurturk** [comment]
Did not know they changed the default region for accounts.

But my problem is different. Now in build.gradle we define already a region and all services are using this configuration. But for create bucket task we have to specify again the region for the second time, if not it suddenly creates in us-west-2. i am sure many people won't even realize this behaviour. I think it should respect to previously set region. Thanks.

**Comment by glebsts** [comment]
Ok, I got your problem. Unfortunately I am not repo maintainer :\
Out of curiosity: before that PR, when you set some non-default region in aws.region, did CreateBucketTask use it?

**Comment by cagataygurturk** [comment]
No, and there was even no region parameter, so it was always creating the bucket in eu-east-something.

**Comment by glebsts** [comment]
>so it was always creating the bucket in eu-east-something

at least now you have a workaround for that, no critical issues. I believe that will be updated some day. Mb some other tasks are no using aws.region as well. Must raise separate issue, mb generic like "fix all tasks that do not use aws.region".
